force-app/main/default/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -282,19 +285,40 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
     }
 
     updateDisplayStateWhenInBatchGiftEntry() {
-        if (this.isPaymentStatusAuthorized() && !this.isPaymentMethodCreditCard()) {
+        if (this.isPaymentStatusAuthorized() && !this.isValidBatchElevatePaymentMethod()) {
             this.display.transitionTo('resetToDeactivated');
         }
 
-        if (this.isPaymentStatusAuthorized() && this.isPaymentMethodCreditCard()) {
+        if (this.isPaymentStatusAuthorized() && this.isValidBatchElevatePaymentMethod()) {
             this.display.transitionTo('readOnly');
-        } else if (this.isPaymentMethodCreditCard()) {
+        } else if (this.isValidBatchElevatePaymentMethod()) {
             this.display.transitionTo('charge');
+            if (this.isMounted) {
+                this.requestSetPaymentMethod(this._currentPaymentMethod);
+            }
         } else {
             this.display.transitionTo('deactivated');
         }
     }
 
+    isValidBatchElevatePaymentMethod() {
+        if (!(this.isPaymentMethodAch() || this.isPaymentMethodCreditCard())) {
+            return false;
+        }
+
+        if (!GeGatewaySettings.getElevateSettings()) {
+            return true;
```

Comment:
At first glance I'm not sure why this is a valid case... feels like we're just missing settings.

Reply:
We are missing settings. Not every template will have elevateSettings. In that case, we do not disable any payment methods.